PowerEditor/src/Notepad_plus.cpp

```diff
@@ -6178,6 +6181,9 @@ void Notepad_plus::notifyBufferActivated(BufferID bufid, int view)
 	}
 
 	_linkTriggered = true;
+	
+	//updating "Go To" dialog
+	_goToLineDlg.updateLinesNumbers();

```

Comment:
You already added this code in `updateStatusBar` which is called just above in this same function.  So this call is unneeded.

Reply:
You're absolutely right, "updateStatusBar()" is called each time a different file tab is clicked. There I've updated it now.